sources/api/netdog/src/wicked/mod.rs

```diff
@@ -35,6 +47,14 @@ pub(crate) struct WickedInterface {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(rename = "ipv6:static")]
     pub(crate) ipv6_static: Option<WickedStaticAddress>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(rename = "vlan")] // huh? not sure about this
```

Comment:
```suggestion
    #[serde(rename = "vlan")]
```
Are we sure about this?

Reply:
I'll remove :)